packages/compass/src/main/telemetry.ts

```diff
@@ -121,6 +121,10 @@ class CompassTelemetry {
     }
   }
 
+  private static _flushTelemetryAndIgnoreFailure() {
+    return CompassTelemetry.analytics?.flush().catch(() => Promise.resolve());
```

Comment:
flush returns a promise now, did i replicate the intended behaviour correctly?

Reply:
I _think_ so but for some reason I'm never 100% sure about how .catch() behaves and have to look it up again every time.

Reply:
I think you did, yes. But just for consistency with the rest of this file:

```suggestion
    return this.analytics?.flush().catch(() => Promise.resolve());
```